### PR TITLE
Predict Remaining Useful Life

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Adds a demo for using Compose to Predict RUL. The demo is located [here](https://compose.featurelabs.com/en/predict-remaining-useful-life/examples/predict-remaining-useful-life/example.html).

### CODE REVIEW

**Comment by kmax12** [comment]
can we add a demonstration of bin labels at the end of this example? that would be cool

**Comment by jeff-hernandez** [comment]
Yes, binned labels at the end of this example. Section can bee seen [here](https://compose.featurelabs.com/en/predict-remaining-useful-life/examples/predict-remaining-useful-life/example.html#Transform-Labels).

File: docs/source/examples/predict-remaining-useful-life/example.ipynb [hunk 1]
```diff
@@ -408,6 +408,59 @@
...
+   ],
+   "source": [
+    "lt.plot.distribution();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that most engines last between 118 to 287 cycles before needing maintenance. With these labels, we can train a machine learning model to predict which range the RUL will fall in."
```
Comment (kmax12) [inline docs/source/examples/predict-remaining-useful-life/example.ipynb]: i think this paragraph is out of place. maybe it i supposed to be on the distribution plot a couple cells above?

Comment (jeff-hernandez) [reply docs/source/examples/predict-remaining-useful-life/example.ipynb]: Moved cells

Comment (jeff-hernandez) [reply docs/source/examples/predict-remaining-useful-life/example.ipynb]: On read the docs, not sure why the distribution plot doesn't show the discrete version. I've tried refreshing, wiping environment, and reactivating version. Locally, works fine.

**Pull request merged**